engine/schema/src/main/java/com/cloud/upgrade/dao/Upgrade41120to41200.java

```diff
@@ -52,7 +57,15 @@ public boolean supportsRollingUpgrade() {
 
     @Override
     public void performDataMigration(Connection conn) {
+        updateManagementServerHostUuid(conn);
+    }
 
+    private void updateManagementServerHostUuid(Connection conn) {
+        try (final PreparedStatement updateStatement = conn.prepareStatement("UPDATE cloud.mshost SET uuid=UUID()")) {
+            updateStatement.executeUpdate();
+        } catch (SQLException e) {
+            LOG.error("Failed to set dynamic.apichecker.enabled to true, please run migrate-dynamicroles.py script to manually migrate to dynamic roles.", e);
```

Comment:
This message looks a little bit misleading, or am I mistaken?

Reply:
Good catch!